dev-utils/karma.js

```diff
@@ -158,7 +164,6 @@ function prepareConfig(config, packageName) {
     console.log('prepareConfig: Run in SauceLab mode')
     config.sauceLabs.build = buildId
     console.log('saucelabs.build:', buildId)
-    config.concurrency = 3
```

Comment:
By setting it to `Infinity`, it starts the process and waits for browser socket connection to run the tests which seems to be work better without hanging to start the browsers after 3 tests passes. https://travis-ci.org/elastic/apm-agent-rum-js/jobs/622655609

Reply:
Doesn't it make the test more unstable in saucelabs since we only have a limited number of sessions there?

Reply:
Nope,Since there is a wait time and also retries involved. Also with this change, I no longer see broken tests due to timeouts in saucelabs. A nice side effect 😄